tests/test_qdrant_version.py

```
@@ -0,0 +1,242 @@
+
+    def test_local_in_memory(self):
+        client = QdrantClient(":memory:")
+        # Local mode returns the client library version, not a server version.
+        result = client.qdrant_version()
+        assert isinstance(result, str)
+        assert result != "<unknown>"
+        # The result should look like a version (e.g. "1.18.0" or "1.18.1-dev")
+        assert any(c.isdigit() for c in result)
+
+    def test_local_path(self):
+        client = QdrantClient(path="/tmp/qdrant_version_test")
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/qdrant_version_test"

(S108)